core/templates/dev/head/pages/collection_player/CollectionPlayer.js

```diff
@@ -275,5 +280,16 @@ oppia.controller('CollectionPlayer', [
         $scope.generatePathParameters();
       }
     }, true);
+
+    $scope.scrollToLocation = function(id) {
+      $location.hash(id);
+      $anchorScroll();
+    };
+    // Touching anywhere outside the mobile preview should hide it.
```

Comment:
So on mobile, why not instead:

When true, lesson card corresponding to the icon clicked is displayed.
When false, no card is displayed.